packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -690,13 +700,15 @@ srusdUsdcPoolMaxDuration = "<%= settings.ONE_MINUTE_IN_SECONDS %>"
 #------------------------------------------------------------------------------
 seliniUsdcReyaLmMaxStaleDuration = "<%= settings.ONE_MINUTE_IN_SECONDS %>"
 amberUsdcReyaLmMaxStaleDuration = "<%= settings.ONE_MINUTE_IN_SECONDS %>"
+hedgeUsdcReyaLmMaxStaleDuration = "<%= settings.ONE_MINUTE_IN_SECONDS %>"
 
 [var.pool_1]
-pool1_quoteTokenTargetRatioUnscaled = "0.3"
+pool1_quoteTokenTargetRatioUnscaled = "0.08"
 pool1_DEUSDTargetRatioPostQuoteUnscaled = "0"
-pool1_SDEUSDTargetRatioPostQuoteUnscaled = "0.7"
-pool1_RSELINITargetRatioPostQuoteUnscaled = "0.15"
-pool1_RAMBERTargetRatioPostQuoteUnscaled = "0.15"
+pool1_SDEUSDTargetRatioPostQuoteUnscaled = "0.744"
```

Comment:
hmm, i think we need to actually bring sdeusd down and rusd up, need to double check with @aaraujoxr and Simon on what ratio we're targeting for rusd (can be done in separate pr if needed)

Reply:
yes, this is a good question but I set these ratios according to the current proportions